docs/getting-started/01-overview/02.hosting.md

```diff
@@ -0,0 +1,144 @@
+Hosting / Execution
+-------------------
+
+## How to host
+==============
+In this section, we'll explore the different ways that Pristine can be hosted.
+
+You can host Pristine under different contexts:
+* Function as a Service (Serverless)
+* NodeJS Server (Container, Virtual machine or your local machine)
+* For scripting purposes by running it directly via the command line
+
+### Function as a Service (Serverless)
+======================================
+
```

Comment:
Maybe a comment saying we want to support Azure and GCP eventually so people don't get scared that we only work with AWS ?